Dockerfile

```diff
@@ -13,25 +14,41 @@
 
 ## Open another bash
 # docker exec -it <CONTAINER_NAME> bash
+
 ## Open another bash with display
 # xhost +local:docker && docker exec -it -e "DISPLAY=${DISPLAY}" <CONTAINER_NAME> bash
 
+
 FROM adamrehn/ue4-build-prerequisites:cudagl10.0
 SHELL ["/bin/bash", "-c"]
 
 USER root
 
+
+# ==============================
+# Configurable params
+# ==============================
+ARG GITHUB_USER
+ARG GITHUB_PWD
+RUN env
```

Comment:
What's this for?

Reply:
This is needed if not dockerfile won't register the passed-in arg using --build-arg. Can remove RUN env though, it is mainly to confirm the ARG are registered.